cdap-cli/pom.xml

```diff
@@ -72,6 +72,10 @@
       <artifactId>concurrent-trees</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.apache.twill</groupId>
```

Comment:
This is needed because ``ProgramId`` has a dependency on Twill's ``RunId`` due to the ``run(RunId runId)`` method.